caniusepython3/test/test_dependencies.py

```diff
@@ -69,6 +69,13 @@ def test_blocking_dependencies_no_project(self):
         got = dependencies.blocking_dependencies(['asdfsadfdsfsdffdfadf'], {})
         self.assertEqual(got, frozenset())
 
+    def test_blocking_dependencies_locators_fails(self):
+        py3 = {'py3_project': ''}
+        breaking_project = 'jekyll2nikola'
```

Comment:
I added

    @mock.patch('distlib.locators.locate')$
     def test_blocking_dependencies_locators_fails(self, distlib_mock):$
         """$
         Testing the work around for //bitbucket.org/pypa/distlib/issue/59/$
         """$
         py3 = {'py3_project': ''}$
         breaking_project = 'test_project'$
         distlib_mock.locators.side_effect = AttributeError()$
         got = dependencies.blocking_dependencies([breaking_project], py3)$
         # If you'd like to test that a message is logged we can use $
         # testfixtures.LogCapture or stdout redirects.

But since other parts of dependencies use distlib.locators.locate() 

======================================================================
ERROR: Testing the work around for //bitbucket.org/pypa/distlib/issue/59/
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.4.2_1/Frameworks/Python.framework/Versions/3.4/lib/python3.4/unittest/mock.py", line 1136, in patched
    return func(*args, **keywargs)
  File "/Users/alexlord/git/caniusepython3/caniusepython3/test/test_dependencies.py", line 81, in test_blocking_dependencies_locators_fails
    got = dependencies.blocking_dependencies([breaking_project], py3)
  File "/Users/alexlord/git/caniusepython3/caniusepython3/dependencies.py", line 119, in blocking_dependencies
    return reasons_to_paths(reasons)
  File "/Users/alexlord/git/caniusepython3/caniusepython3/dependencies.py", line 43, in reasons_to_paths
    parent = reasons[blocker]
  File "/Users/alexlord/git/caniusepython3/caniusepython3/dependencies.py", line 29, in __getitem__
    return super(LowerDict, self).__getitem__(key.lower())
nose.proxy.KeyError: <MagicMock name='locate().name.lower().lower()' id='4345929400'>
-------------------- >> begin captured logging << --------------------
ciu: INFO: Checking top-level project: test_project ...
ciu: INFO: Locating <MagicMock name='locate().name.lower()' id='4344734944'>
ciu: INFO: Dependencies of <MagicMock name='locate().name.lower()' id='4344734944'>: []
--------------------- >> end captured logging << ---------------------